ntia-core.sigmf-ext.md

```diff
@@ -25,6 +25,8 @@ The Measurement object summarizes the basic measurement information including  w
 |`frequency_tuned_high`|true|double|Hz|Highest tuned  frequency.|
 |`frequency_tuned_step`|false|double|Hz|Step between tuned frequencies of a `"scan"` measurement. Either `frequency_tuned_step` or `frequencies_tuned` SHOULD be included for `"scan"` measurements.|
 |`frequencies_tuned`|false|double[]|Hz|Array of tuned frequencies of a `"scan"` measurement. Either `frequency_tuned_step` or `frequencies_tuned` SHOULD be included for `"scan"` measurements.|
+|`classification`|true|string|N/A|The classification markings for the acquisition.|
```

Comment:
Remove leading "The". Append with "Either `UNCLASSIFIED`, `SENSITIVE`, `CLASSIFIED`.